types/jquery-cropper/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for jquery-cropper v1.0.1
+// Project: https://github.com/davidstutz/jquery-cropper
```

Comment:
This link doesn't work

Reply:
the correct link is:
https://github.com/fengyuanchen/jquery-cropper